SB.js

```diff
@@ -0,0 +1,97 @@
+SB = {};
+
+Map.prototype.toJSON = function() {
+    return Array.from(this.entries());
+};
+
+function storeEncode(data) {
+	if(!(data instanceof Map)) return data;
+	return JSON.stringify(data);
+}
+
+function strParser(data) {
+	try {
+        return new Map(JSON.parse(data));
+    } catch(e) {
+        return data
+    }
+}
```

Comment:
Change try catch to a type check maybe? Check for type array?

Reply:
Well I think it would be a string.

Reply:
JSON.parse would return a array